h2o-r/h2o-package/R/models.R

```diff
@@ -2547,15 +2551,15 @@ h2o.coef_with_p_values <- function(object) {
 #' }
 #' @export
 h2o.get_variable_inflation_factors <- function(object) {
-  if (is(object, "H2OModel") && object@algorithm %in% c("glm")) {
-    if (object@parameters$generate_variable_inflation_factors) {
-      structure(object@model$variable_inflation_factors, names = object@model$vif_predictor_names)
+    if (is(object, "H2OModel") && object@algorithm %in% c("glm")) {
+        if (object@parameters$generate_variable_inflation_factors) {
+            structure(object@model$variable_inflation_factors, names = object@model$vif_predictor_names)
+        } else {
+            stop("variable inflation factors are not found in model.  Make sure to set enable_variable_inflation_factors=TRUE.")
+        }
     } else {
-      stop("variable inflation factors are not found in model.  Make sure to set enable_variable_inflation_factors=TRUE.")
+        stop("variable inflation factors are only found in GLM models with numerical predictors.")
```

Comment:
Please add

Make sure to set enable_variable_inflation_factors=TRUE.

Reply:
The PR is still a draft and not rebased on the actual master. So after that, this will be resolved.